forte/data/data_store.py

```diff
@@ -188,10 +195,24 @@ def __init__(self, onto_file_path: Optional[str] = None):
         A dictionary that keeps record of all entrys with their tid.
         It is a key-value map of {tid: entry data in list format}.
 
-        e.g., {1423543453: [begin, end, tid, type, attr_1, ..., attr_n]}
+        e.g., {1423543453: [begin, end, tid, entry_type, attr_1, ..., attr_n],
+        4345314235: [parent_tid, child_tid, tid, entry_type, attr_1, ...,
+                    attr_n, index_id]}
         """
         self.__entry_dict: dict = {}
 
+    def _new_tid(self) -> int:
+        r"""This function generates a new `tid` for an entry."""
+        return uuid.uuid4().int
+
+    def _get_entry_type_str(
```

Comment:
Consider implement this as a general utility method? Maybe here: https://github.com/asyml/forte/blob/master/forte/utils/utils.py

Also check that file to see if any utilities can be reused?

Reply:
I just found a similar function in utils. I will import that function directly and remove _get_entry_type_str.